Carbon.Registry/Functions/Get-CRegistryPermission.ps1

```diff
@@ -0,0 +1,88 @@
+
+function Get-CRegistryPermission
+{
+    <#
+    .SYNOPSIS
+    Gets the permissions (access control rules) for a file, directory, registry key, or certificate's private key/key

```

Comment:
Is this supposed to mention file, directory, and certificate paths (i.e. non registry paths)?